CONTRIBUTING.md

```diff
@@ -28,6 +28,15 @@ cabal test containers-test
 cabal bench containers-test
 ```
 
+#### For Windows users : 
+
+To compile `containers-test`, you need symbolic links to be activated on git.
+To do so on Windows 10 on higher, follow these steps :
+
+1. Activate developper mode on in your Windows preferences
+2. Enable git symlinks : `git config --global core.symlinks true`
+3. Clone the repository again once git is properly configured
```

Comment:
```suggestion
To compile `containers-test`, you need symbolic links to be activated on git.
To do so on Windows 10 or higher, follow these steps:

1. Activate developer mode in your Windows preferences
2. Enable git symlinks: `git config --global core.symlinks true`
3. Clone the repository again once git is properly configured
```

Reply:
Let's put this in a separate Windows section so people running Linux, OSX, etc., don't get confused.